components/NavBar/index.js

```diff
@@ -1,34 +1,70 @@
 import styled from "styled-components";
 import Link from "next/link";
+import { useRouter } from "next/router";
 
 export default function NavBar() {
+  const router = useRouter();
+
   return (
     <StyledNavigation>
-      <StyledLink href="/">
-        <img src="/Images/icons8-startseite-50.png" />
-      </StyledLink>
-      <StyledLink href="/WatchList">
-        <img src="/Images/icons8-lesezeichen-50.png" />
-      </StyledLink>
+      <StyledLink1 active={router.asPath === "/"} href="/" className="active">
```

Comment:
There seems to be an old relic here 🔍 
```suggestion
      <StyledLink1 active={router.asPath === "/"} href="/">
```